docs/src/main/sphinx/optimizer/pushdown.md

```diff
@@ -24,7 +24,7 @@ typically resulting from a condition in a `WHERE` clause to omit unnecessary
 rows. The processing is pushed down to the data source by the connector and then
 processed by the data source.
 
-If predicate pushdown for a specific clause is succesful, the `EXPLAIN` plan
+If predicate pushdown for a specific clause is successful, the `EXPLAIN` plan
```

Comment:
Could you fix the same typo in `json.md`?

Reply:
@emilysunaryo ?

Reply:
You can just do a global search for that typo and fix all of them in this PR